tests/framework/test_engine.py

```diff
@@ -42,40 +47,39 @@
 from vivarium.framework.values import ValuesInterface, ValuesManager
 
 
-def is_same_object_method(m1, m2):
+def is_same_object_method(m1: MethodType, m2: MethodType) -> bool:
```

Comment:
If this method takes a more generic type and you do the casting inside here, you can probably remove some duplicated code elsewhere in this file.

Reply:
good idea.